cmd/swarm/config.go

```diff
@@ -259,6 +259,10 @@ func cmdLineOverride(currentConfig *bzzapi.Config, ctx *cli.Context) *bzzapi.Con
 		currentConfig.LocalStoreParams.CacheCapacity = storeCacheCapacity
 	}
 
+	if ctx.Bool(SwarmDisableCacheFlag.Name) {
```

Comment:
@justelad can't we just use the value 0 to disable it - why do we need another flag?

Reply:
if you omit the cache size flag - the check in the block above will yield that the value is `0`. that's why `0` value is ignored in the if statement. we can maybe change it to be a string flag and try to parse the string into an int. WDYT?

Reply:
Why not to use `ctx.IsSet` instead to check if the value is 0?

Reply:
you're right @janos. making the change now. the CLI library documentation does not explicitly mention this feature, just by looking at the examples :/